redoing: namespace get_template_part action hook

cliffordp opened the pull request:
by adding a new action hook instead of replacing the existing one

replacement for the pull request that _was_ at https://github.com/GaryJones/Gamajo-Template-Loader/pull/25

cliffordp committed:
redoing: namespace get_template_part action hook

by adding a new action hook instead of replacing the existing one

replacement for the pull request that _was_ at https://github.com/GaryJones/Gamajo-Template-Loader/pull/25

GaryJones commented on class-gamajo-template-loader.php:
```diff
@@ -110,6 +110,7 @@ public function __destruct() {
 		public function get_template_part( $slug, $name = null, $load = true ) {
 			// Execute code for this part.
 			do_action( 'get_template_part_' . $slug, $slug, $name );
+			do_action( 'get_template_part_' . $this->filter_prefix . '_' . $slug, $slug, $name );
```
I think it should be:
```php
do_action( $this->filter_prefix . '_get_template_part_' . $slug, $slug, $name );
```
as that's how the `filter_prefix` property is used in two existing filters.

cliffordp committed:
Make requested change

= Example =
Was: `get_template_part_tk_event_weather_hourly_horizontal`
Is now: `tk_event_weather_get_template_part_hourly_horizontal`

GaryJones commented:
Thank you :-)

cliffordp commented:
Thank _you_!

Also wanted to remind you of the version bump in both the file's comments and the tag on github.

GaryJones commented:
@cliffordp Thanks for the nudge :-)

https://github.com/GaryJones/Gamajo-Template-Loader/releases/tag/1.3.0
